README.md

```diff
@@ -74,15 +74,115 @@ sequenceDiagram
 
 ### Requirements
 
-- A Kubernetes cluster
+- A Kubernetes cluster (GKE, AKS, etc.)
 - An API token with the [GraphQL scope enabled](https://buildkite.com/docs/apis/graphql-api#authentication)
 - An [agent token](https://buildkite.com/docs/agent/v3/tokens)
+- A Cluster UUID (if using [Buildkite Clusters](https://buildkite.com/docs/agent/clusters))
 
-### Deploy with Helm
 
-The simplest way to get up and running is by deploying our [Helm](https://helm.sh) chart:
+### Creating the K8's cluster
```

Comment:
[grammar pedantry??] "K8s" is a contraction of "Kubernetes", not a possessive, so:

```suggestion
### Creating the K8s cluster
```